internal/pkg/task/env_runner.go

```diff
@@ -156,3 +157,48 @@ func containsString(s []string, search string) bool {
 	}
 	return false
 }
+
+// CheckNonZeroExitCode returns the status of the containers part of the given tasks.
+func (r *EnvRunner) CheckNonZeroExitCode(tasks []*Task) error {
```

Comment:
I feel this function should be placed in the `ecs` package instead. Can we move this function to https://github.com/aws/copilot-cli/tree/507747027c5ff3007b3c99e1f9c33aad74a63566/internal/pkg/ecs? And then we can call it in both config runner and env runner to reduce code duplication?

Reply:
I have moved the common functionality to the `ecs` package